src/navigation/navigationTabs/__test__/tabs.test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * To run this test from the root folder, execute the following command:
+ * npx jest ./src/navigation/navigationTabs/__test__/tabs.test.js
+ */
+// import { shallow } from 'enzyme';
```

Comment:
Comments will be removed leaving the actual test cases after resolving the issue with the `docs generator` and multiple file exports.

Reply:
@War777 please see #194 specifically 058389b.  I was also having similar troubles.  You _should_ be able to test everything you need only exporting the component wrapped in HOC(s), using `mountWithTheme()` helper method (rather than `shallow()`), and then using `find` to grab the inner component and access its `instance()`.  Let me know if I can be of further assistance.

Reply:
Unit test fixed usint `mountWithTheme()`